docs/getting-started/creating-a-new-application.md

```diff
@@ -4,4 +4,43 @@ title: Creating a new application
 sidebar_label: Create a new application
 ---
 
-## TODO
\ No newline at end of file
+## Prerequisites
+
+Before you start, check to make sure you have the following installed:
+
+- [Node.js](https://nodejs.org/en/) v10.15.x or later
+- [npm](https://www.npmjs.com/) v5.x or later
```

Comment:
yarn :)

Reply:
I left it out because yarn users know how to use yarn and npm, but not the other way around.

A yarn user will automatically know "I can do this in yarn". It is optional to use yarn so I did not want to have it as a requirement, which would really alienate non-yarn users.